radix-engine/assets/native_function_base_costs.csv

```diff
@@ -201,6 +201,9 @@
 0d906318c6318c6e8f9fcc0c6318c6318cf7aa2fad74a29e26318c6318c6,lock_owner,610324
 0d906318c6318c6e8f9fcc0c6318c6318cf7aa2fad74a29e26318c6318c6,set,925129
 0d906318c6318c6e8f9fcc0c6318c6318cf7aa2fad74a29e26318c6318c6,set_owner,548852
+0d906318c6318c6ee313598c6318c6318cf7bcaa2e954a9626318c6318c6,balance,1402247
```

Comment:
Please do not merge until this is updated. I have CI running to calculate what the values should be for those.

Reply:
Done.